src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -3662,7 +3662,7 @@ INSERT INTO tab_calculation_method_rates VALUES('DistributionInVitro_PKSim','Bin
 INSERT INTO tab_calculation_method_rates VALUES('DistributionInVitro_PKSim','Binding partner unknown','2','Dimensionless');
 INSERT INTO tab_calculation_method_rates VALUES('DistributionInVitro_PKSim','One_Rate','1','Dimensionless');
 INSERT INTO tab_calculation_method_rates VALUES('DistributionInVitro_PKSim','PARAM_Blood2Plasma','(f_water_rbc + f_lipids_rbc * 10^LogP + f_proteins_rbc * KProt) * fu * HCT - HCT + 1','Dimensionless');
-INSERT INTO tab_calculation_method_rates VALUES('DistributionInVitro_PKSim','PARAM_fu','1 / (1 + (BindingPartner = Albumin ? AlbuminOntgenyFactor : (BindingPartner = Agp ? AgpOntogenyFactor : 1) ) * (1 - fu_ref) / fu_ref) * PlasmaProteinScaleFactor','Fraction');
+INSERT INTO tab_calculation_method_rates VALUES('DistributionInVitro_PKSim','PARAM_fu','Min(1 / (1 + (BindingPartner = Albumin ? AlbuminOntgenyFactor : (BindingPartner = Agp ? AgpOntogenyFactor : 1) ) * (1 - fu_ref) / fu_ref) * PlasmaProteinScaleFactor; 1)','Fraction');

```

Comment:
@Yuri05 This was an error that we had previously. High fu and high factor would result in fu >1 and the simulation barfing